src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -168,6 +169,28 @@ _ResolveAssemblies MSBuild target.
         IncludeDebugSymbols="$(AndroidIncludeDebugSymbols)">
       <Output TaskParameter="OutputLibraries" ItemName="FrameworkNativeLibrary" />
     </ProcessNativeLibraries>
+    <ItemGroup>
+      <LocalFrameworkNativeLibrary Include="$(IntermediateOutputPath)native\%(FrameworkNativeLibrary.RuntimeIdentifier)\%(FrameworkNativeLibrary.Filename)%(FrameworkNativeLibrary.Extension)">
+        <RuntimeIdentifier>%(FrameworkNativeLibrary.RuntimeIdentifier)</RuntimeIdentifier>
+      </LocalFrameworkNativeLibrary>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_StripFrameworkNativeLibraries"
+      Condition=" '$(AndroidIncludeDebugSymbols)' != 'true' "
+      Inputs="@(FrameworkNativeLibrary)"
+      Outputs="@(LocalFrameworkNativeLibrary)">
+    <StripNativeLibraries
+        Libraries="@(FrameworkNativeLibrary)"
+        LocalPath="$(IntermediateOutputPath)native"
+        AndroidBinUtilsDirectory="$(AndroidBinUtilsDirectory)">
+      <Output TaskParameter="OutputLibraries" ItemName="StrippedFrameworkNativeLibrary" />
```

Comment:
same for `StrippedFrameworkNativeLibrary`. it should be internal `_ StrippedFrameworkNativeLibrary`. Unless we expect users to add items to this item group?

Reply:
Instead of making this an `<Output/>` at all, could we pass in `@(LocalFrameworkNativeLibrary)` as an input to be used for the destination files?

Then I think this target would be able to build partially.